main.pm

```diff
@@ -64,10 +64,10 @@ sub load_shutdown() {
 }
 
 # load tests in the right order
-if (get_var('UPDATE')) {
+if (check_var('MODE', 'update') or get_var('UPDATE')) {
```

Comment:
Looks like "update" is supposed to be the default. So I guess this should be:
```suggestion
if (check_var('MODE', 'update') || check_var('MODE', '')) {
```
I'd also prefer `||` in conditionals (and only use `or` for command-chaining).

Reply:
I think you are confusing the use of the second argument of `check_var` here? `check_var('MODE', 'update')` returns true if MODE is set to "update". The backward-alternative evaluation is `get_var('UPDATE')` that evaluates to true if UPDATE is set to a truthy value.

Reply:
I was thinking about the case when none of the variables is set and wrote my suggestion so that in this case the update-path is executed. I guess executing none of the special paths is fine as well.